src/DetailsView/components/failure-instance-panel-control.tsx

```diff
@@ -135,29 +137,37 @@ export class FailureInstancePanelControl extends React.Component<FailureInstance
                 />
                 <TextField
                     className="observed-failure-textfield"
-                    label="Comments"
+                    label="Comment"
+                    styles={getStyles}
                     multiline={true}
-                    rows={8}
+                    rows={4}
                     value={this.state.currentInstance.failureDescription}
                     onChange={this.onFailureDescriptionChange}
                     resizable={false}
                 />
+            </GenericPanel>
+        );
+    }
+
+    private getActionCancelButtons = (): JSX.Element => {
+        return (
+            <div className="footer">
                 <ActionAndCancelButtonsComponent
                     isHidden={false}
                     primaryButtonDisabled={
                         this.state.currentInstance.failureDescription === null && this.state.currentInstance.path === null
                     }
-                    primaryButtonText={this.props.actionType === CapturedInstanceActionType.CREATE ? 'Add' : 'Save'}
+                    primaryButtonText={this.props.actionType === CapturedInstanceActionType.CREATE ? 'Add failed instance' : 'Save'}
```

Comment:
I've noticed you're checking for this "Create" action type twice; you could isolate this, at least visually, by putting them in local variables (that you reference here) in an if statement before the return statement.

Reply:
Got it! Created a new Boolean variable called createMode

Reply:
Boolean in and tests are passing! Thank you!